README.md

```diff
@@ -1,2 +1,40 @@
-# aquality-appium-mobile-java-template
-Template project for aquality-appium-mobile-java library.
+[![Build Status](https://dev.azure.com/aquality-automation/aquality-automation/_apis/build/status/aquality-automation.aquality-appium-mobile-java-template?branchName=master)](https://dev.azure.com/aquality-automation/aquality-automation/_build/latest?definitionId=11&branchName=master)
+[![Quality Gate](https://sonarcloud.io/api/project_badges/measure?project=aquality-automation_aquality-appium-mobile-java-template&metric=alert_status)](https://sonarcloud.io/dashboard?id=aquality-automation_aquality-appium-mobile-java-template)
+
+# Aquality Appium Mobile Template Project
+Template for [aquality-appium-mobile-java](https://github.com/aquality-automation/aquality-appium-mobile-java) library.
+
+### Project structure
+- **aquality-appium-mobile-template** - project related part with PageObjects, models and utilities
+  - **configuration/**: classes that used to fetch project config from [src/main/resources/environment](https://github.com/aquality-automation/aquality-appium-mobile-java-template/blob/master/aquality-appium-mobile-template/src/main/resources/environment) folder
+  - **screens/**: Page Objects
+  - **models/**: classes that represent data models of the application under the test (POJO classes)
+  - **utilities/**: util classes
+  - **src/main/resources/**: resource files such as configurations and test data
+- **aquality-appium-mobile-template-cucumber** - Cucumber implementation of the tests
+  - **features/**: Gherkin feature files with test scenarios
+  - **hooks/**: Cucumber [hooks](https://specflow.org/documentation/Hooks/)
+  - **stepdefinitions/**: step definition classes
+  - **transformations/**: Cucumber [data transformations](https://cucumber.io/docs/cucumber/configuration/)
+  - **objectfactory/**: [DI container](https://cucumber.io/docs/cucumber/state/#how-to-use-di) configuration
+
+### Configuration
+[settings.json](https://github.com/aquality-automation/aquality-appium-mobile-java-template/blob/master/aquality-appium-mobile-template/src/main/resources/settings.json) file contains settings of Aquality Appium Mobile library. Additional information you can find [here](https://github.com/aquality-automation/aquality-selenium-java/wiki/Overview-(English)).
```

Comment:
Is the link to https://github.com/aquality-automation/aquality-selenium-java/wiki/Overview-(English) correct?

Reply:
The concepts of settings is OK, but we don't have docs in appium library.